src/app/chips/chips.sample.html

```diff
@@ -4,7 +4,6 @@
         <div class="sample-holder">
             <igx-chips-area #chipsArea (onReorder)="chipsOrderChanged($event)" (onMoveEnd)="chipMovingEnded()" (onSelection)="onChipsSelected($event)">
                 <igx-chip *ngFor="let chip of chipList" [id]="chip.id" [draggable]="true" [removable]="true" [selectable]="true" (onRemove)="chipRemoved($event)">
-                    <igx-icon igxPrefix fontSet="material" [name]="'drag_indicator'"></igx-icon>
```

Comment:
Is the drag indicator going away?

Reply:
since chips are draggable always by design Spas insisted on removing the indicator to simplify the look of the chip and I support his reasons for doing so

Reply:
Yes, we don't need drag indicator, the chip it's expected to be a draggable element.